1.9/storage/external-storage.md

```diff
@@ -136,7 +136,7 @@ Below is a sample app definition that uses a Docker container and specifies an e
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
I believe this should be 
```
mkdir /data/test-rexray-volume/test-dir && touch /data/test-rexray-volume/test-dir/test.txt && ls /data/test-rexray-volume/test-dir/
```

The directory path in the container is given by the `containerPath` property, which in this case is `/data/test-rexray-volume`. Note that it is an absolute path in this case, as this is using the Docker containerizer, which expects absolute paths for `containerPath` and mounts the external volume at exactly the given path in the container.